backend/src/nodes/nodes/ncnn/upscale_image.py

```diff
@@ -69,16 +70,24 @@ def __init__(self):
     def upscale(
         self,
         img: np.ndarray,
-        net,
+        model: NcnnModel,
         input_name: str,
         output_name: str,
-        max_tile_size: Union[int, None],
+        tile_size: Union[int, None],
```

Comment:
It's called max tile size, cause that's what it is. auto_split will choose tiles that are at most this large, not exactly this large.

Reply:
Ahh, whatever.

Reply:
We can just change it back later. Sorry but I can't change it right this second

Reply:
Nah, it's fine. We'll likely have to change that variable later anyway, and it's not important enough to hold the PR.